nginx/datadog_checks/nginx/data/conf.yaml.example

```diff
@@ -82,6 +82,12 @@ instances:
     #
     # plus_api_version: <PLUS_API_VERSION>
 
+    ## @param only_query_enabled_endpoints - boolean - optional - default: false
+    ## Set this option if you would like the check to determine which API endpoints
+    ## to query based on your NGINX Plus configuration.
+    #
+    only_query_enabled_endpoints: true
```

Comment:
If it isn't set then it defaults to false [here](https://github.com/DataDog/integrations-core/blob/master/process/datadog_checks/process/data/conf.yaml.example#L52). this is so that users who upgrade the agent don't have a behavior change, but new users will have the option enabled by default.

Reply:
Ah got it. Thanks!